src/interfaces/packagingInterfacesAndType.ts

```diff
@@ -37,4 +42,31 @@ export interface IPackage2Version {
 export type PackageOptions = Record<string, unknown>;
 
 export type PackageVersion2Options = Record<string, unknown>;
-export type PackageVersion1Options = Record<string, unknown>;
+export type PackageVersionOptions1GP = Record<string, unknown>;
+
+export type Package2VersionCreateRequestResult = {
+  Id: string;
+  Status: Package2VersionStatus;
+  Package2Id: string;
+  Package2VersionId: string;
+  SubscriberPackageVersionId: string | null;
+  Tag: string;
+  Branch: string;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  Error: any[];
+  CreatedDate: string;
+  HasMetadataRemoved: boolean | null;
+  CreatedBy: string;
+};
+
+export type Package2VersionCreateRequestError = {
+  Message: string;
+};
+
+export type Package2VersionCreateEventData = {
```

Comment:
To be used for Lifecycle events during long running package version create operations.